packages/components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@quri/squiggle-components",
-  "version": "0.8.6",
+  "version": "0.8.7-0",
```

Comment:
0.8.7-0?

Reply:
That's how it should be, it's versions for the next release.